### DISCUSSION TITLE
Tokio for windows?

### DISCUSSION THREAD

**Discussion by sidShinde** [tokio-rs/tokio#4946] [Q&A] [answered]
Hi, I am new to RUST programming and to this community. I couldn't find windows platform support on the tokio docs page for the newer releases. I believe the last release with windows support is v1.9.0. Are there any plans to add windows support in future releases? If yes, is there a tentative release scheduled? If not, would you recommend building it from source? Thank you in advance!

**Comment by Darksonn** [comment, marked as answer]
All 1.x.y releases of Tokio support windows.

**Reply by sidShinde** [reply]
Got it. So for my understanding even if the platforms drop-down on this [page](https://docs.rs/tokio/latest/tokio/index.html) does not indicate windows support explicitly I can still use it as a dependency on windows?

**Reply by Darksonn** [reply]
That dropdown has nothing to do with which platforms that the library is supported on. Instead, it shows which platform the `cargo doc` command was called on.

We do not generate documentation on Windows because we have configured our Windows-specific documentation to show up even when running `cargo doc` on Linux.

**Reply by sidShinde** [reply]
Understood. Thank you @Darksonn!
